meeting-notes/2018-03-05-js-ipfs-dev-team-weekly.md

```diff
@@ -0,0 +1,115 @@
+# ⒿⓈ⚡️ js-ipfs Core Dev Team Weekly 2018-03-05
+
+- **Lead:** Volker Mische (@vmx)
+- **Notetaker:** dryajov
+- **Attendees:**
+  - @victorbjelkholm
+  - @mkg20001
+  - @jonkrone
+  - @dryajov
+  - @zcstarr
+- **Recording:** https://ipfs.io/ipfs/QmdZ1UXtUWuBirybDSuVtxXaZDAi1A1e1KNLyVgcBXtebd
```

Comment:
@dryajov please the youtube url for consistency (or at least add both). Check the issue for urls